### PR TITLE
ENH: Support and/or expressions

### PR DESCRIPTION
**Pull request by ssanderson**

### CODE REVIEW

File: codetransformer/decompiler.py [hunk 1]
```diff
@@ -107,6 +115,29 @@ def _process_jump(instr, queue, stack, body, context):
         )
 
 
+# @_process_instr.register(instrs.JUMP_IF_TRUE_OR_POP)
```
Comment (llllllllll) [inline codetransformer/decompiler.py]: why is this all commented out?

File: codetransformer/utils/functional.py [hunk 2]
```diff
@@ -87,3 +89,24 @@ def ffill(iterable):
         else:
             previous = e
             yield e
+
+
+def moving_window(n, iterable):
```
Comment (llllllllll) [inline codetransformer/utils/functional.py]: This is just `toolz.itertoolz.sliding_window`

File: codetransformer/decompiler.py [hunk 3]
```diff
@@ -1145,6 +1175,57 @@ def _handler(toplevel, stack_builders):
     _make_expr.register(instrtype)(_binop_handler(nodetype))
 
 
+def _make_expr(toplevel, stack_builders, _real_make_expr=_make_expr):
```
Comment (llllllllll) [inline codetransformer/decompiler.py]: should we just rename the old make function? If I try to add a dispatch case after this it won't work

File: codetransformer/decompiler.py [hunk 4]
```diff
@@ -14,6 +14,11 @@
 from codetransformer import a as showa, d as showd  # noqa
 
 
+def _current_test():
```
Comment (llllllllll) [inline codetransformer/decompiler.py]: could the docstring exmplain that this is for debugging only or something?

File: codetransformer/decompiler.py (hunk 1 above)
Comment (ssanderson) [inline codetransformer/decompiler.py]: This was an aborted implementation that I was looking at for reference.  Removed.

File: codetransformer/utils/functional.py (hunk 2 above)
Comment (ssanderson) [inline codetransformer/utils/functional.py]: Updated.

**Comment by llllllllll** [comment]
Other than the docstring and the comment about potentially renaming the _real_make_expr this looks great.

**Pull request merged**